execution_workflows/diffUTR/diffUTR_nextflow/results/pipeline_info/pipeline_dag.svg

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Can you copy the dag into the `diffUTR_nextflow` directory and include it in the `README`? We do that for the snakemake template and would also be nice here!    
Agree with Dominik that the rest of the results should not be included in the repo.

Reply:
Oops I just deleted it...